pytorch/docker-compose.yaml

```diff
@@ -69,28 +69,28 @@ services:
     network_mode: host
     ports:
       - 8888:8888
-  multinode:
-    build:
-      labels:
-        dependency.apt.gcc: true
-        dependency.apt.libgl1-mesa-glx: true
-        dependency.apt.libglib2: true
-        dependency.apt.python3-dev: true
-        dependency.pip.apt.virtualenv: true
-        dependency.pip.deepspeed: 0.14.4
-        dependency.python.pip: multinode/requirements.txt
-        org.opencontainers.base.name: "intel/intel-optimized-pytorch:${IPEX_VERSION:-2.4.0}-${PACKAGE_OPTION:-pip}-base"
-        org.opencontainers.image.title: "Intel® Extension for PyTorch MultiNode Image"
-        org.opencontainers.image.version: ${IPEX_VERSION:-2.4.0}-${PACKAGE_OPTION:-pip}-multinode
-      target: multinode
-    command: >
-      bash -c "python -c 'import neural_compressor;import oneccl_bindings_for_pytorch as oneccl;import deepspeed;
-      print(\"Neural Compressor:\", neural_compressor.__version__,
-      \"\\nOneCCL:\", oneccl.__version__,
-      \"\\nDeepspeed:\", deepspeed.__version__)'"
-    extends: ipex-base
-    image: ${REGISTRY}/${REPO}:b-${GITHUB_RUN_NUMBER:-0}-${BASE_IMAGE_NAME:-ubuntu}-${BASE_IMAGE_TAG:-22.04}-${PACKAGE_OPTION:-pip}-py${PYTHON_VERSION:-3.10}-ipex-${IPEX_VERSION:-2.4.0}-oneccl-inc-${INC_VERSION:-3.0.2}
-    shm_size: 2gb
+  # multinode:
```

Comment:
why not remove instead of commenting?

Reply:
This spec can be re-used for when we create versions for other accelerators.

Reply:
Additionally, if a user wants to create a newer version of this image, they can just uncomment rather than search our tag history.

Reply:
Should we just delete?

Reply:
lets delete it. we can always refer back if needed